crates/bevy_core/src/name.rs

```diff
@@ -76,6 +78,25 @@ impl std::fmt::Display for Name {
     }
 }
 
+/// Convenient query for giving a human friendly name to an entity.
```

Comment:
Can you add a `no_run` doc test here to demonstrate how this is used?

Reply:
Actually helped me catch that I implemented Debug on DebugName instead of DebugNameItem, should we keep it no_run?

Reply:
Nah, it's fine to let it run.